hacep/src/main/java/org/kie/hacep/consumer/CommandHandler.java

```diff
@@ -253,6 +261,28 @@ public void visit(SnapshotOnDemandCommand command) {
         }
     }
 
+    @Override
+    public void visit(UpdateKJarCommand command) {
+        KieServices ks = KieServices.get();
+        if (ks != null) {
+            logger.info("Updating KieContainer with KJar:{}", command.getKJarGAV());
+            ReleaseId releaseId = ks.newReleaseId(command.getGroupID(), command.getArtifactID(), command.getVersion());
+            try {
+                kieSessionContext.getKieContainer().updateToVersion(releaseId);
```

Comment:
What about prevent the call using env.isUpdatableKjar() instead of try and fail if not supported?

Reply:
ok

Reply:
I'll add the check but since if the env.IsUpdatableKjar is just the presence of the env is more safe if we leave even the try catch in place